### DISCUSSION TITLE
Is Semantic Kernel ready to be used in production?

### DISCUSSION THREAD

**Discussion by efonsecab** [microsoft/semantic-kernel#6475] [Q&A] [answered]
I've executed some tests with Semantic Kernel, and it has met some of my needs.
I've been considering if replacing my existent AI Services class with Semantic Kernel, however, something I have seen is that many features are marked as experimental.
So I'd like to know when it is expected to be completely ready for production: for me it means, the main branch APIs having all graduated from experimental.

**Comment by stephentoub** [comment, marked as answer]
FWIW, [Experimental] isn't a statement here on quality but rather an acknowledgement that the API might change in the future. You can use such features in production, it's just that in the future when you upgrade your version, there may be breaking changes you'll need to factor in, e.g. the functionality exposed in a different way.
